spec/fixtures/_setup.scss

```diff
@@ -0,0 +1,2 @@
+@import "../../core/neat";
```

Comment:
I can’t say that I know how or why this works, but I think this can just be `@import "core/neat";`. At least, it seems to work in Bourbon: https://github.com/thoughtbot/bourbon/blob/master/spec/fixtures/_setup.scss